analyzers/its/Directory.Build.targets

```diff
@@ -5,8 +5,9 @@
     <PropertyGroup>
       <BinariesFolder>$(MSBuildStartupDirectory)\..\packaging\binaries</BinariesFolder>
       <AnalysisOutputPath>$(MSBuildStartupDirectory)\output\$(PROJECT)\$(AssemblyName)</AnalysisOutputPath>
-      <ErrorLogPrefix Condition="'$(TargetFramework)' == ''">$(MSBuildStartupDirectory)\output\$(PROJECT)\$(AssemblyName)</ErrorLogPrefix>
-      <ErrorLogPrefix Condition="'$(TargetFramework)' != ''">$(MSBuildStartupDirectory)\output\$(PROJECT)\$(AssemblyName)-$(TargetFramework)</ErrorLogPrefix>
+      <ErrorLogFolder>$(MSBuildStartupDirectory)\output\Issues\$(PROJECT)</ErrorLogFolder>
```

Comment:
Need to extract this so that the directory exists when the analyzers write to write the SARIFs.